app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java

```diff
@@ -362,4 +370,23 @@ public void run() {
 
         });
     }
-}
\ No newline at end of file
+
+    @Override
+    public void onCesiumEntityClick(String layerId, String entityId) {
```

Comment:
I'd split this method into validation (which throws an exception), entity parsing, and clicking